nautobot_golden_config/templates/nautobot_golden_config/compliance_device_report.html

```diff
@@ -170,6 +170,19 @@ <h1>{% block title %}Configuration Compliance - {{ device.name }}{% endblock %}<
                 </td>
             </tr>
         {% endif %}
+        {% if item.remediation != None %}
+        <tr>
+            <td style="color:red;width:250px">Remediating Configuration</td>
+            <td class="config_hover">
+                <span id="{{ item.rule|slugify }}_remediation"><pre>{{ item.remediation|condition_render_json }}</pre></span>
+                <span class="config_hover_button">
+                    <button class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#{{ item.rule|slugify }}_remediation">
+                        <span class="mdi mdi-content-copy"></span>
+                    </button>
+                </span>
+            </td>
+        </tr>
+    {% endif %}
```

Comment:
```suggestion
        {% endif %}
```

Just fixing for my OCD.